src/page-multi-facility/ScenarioLibraryModal.tsx

```diff
@@ -23,7 +23,7 @@ const ModalContents = styled.div`
   font-weight: normal;
   justify-content: flex-start;
   margin-top: 30px;
-  height: 100%;
+  height: auto;
```

Comment:
This causes a subtle UI degradation that probably isn't worth worrying about, but that we should at least be aware of.  The degradation is that the `Loading` icon is now shifted somewhat awkwardly to the top of the modal during a transition state (i.e. when we're duplicating a scenario).

**Before (height: 100%)**

<img width="763" alt="Screen Shot 2020-06-09 at 9 46 27 AM" src="https://user-images.githubusercontent.com/25503/84155629-8dab6f80-aa36-11ea-94a4-b00204579ec0.png">

**Before (height: auto)**

<img width="762" alt="Screen Shot 2020-06-09 at 9 47 26 AM" src="https://user-images.githubusercontent.com/25503/84155657-9603aa80-aa36-11ea-986b-72dd39c7dc2c.png">

The ideal fix for the original issue would be one that didn't also cause this minor UI degradation.  I probably wouldn't spend much time on it, but it's likely worth timeboxing an investigation to see if we can get the best of both worlds.

Reply:
great catch. I've made a few changes in CSS and DOM structure around where we render the loading component. Please take another look. Thanks!